### DISCUSSION TITLE
WebServerFreeRTOS on Nucleo F746ZG board hangs after few minutes

### DISCUSSION THREAD

**Discussion by xRadx** [stm32duino/Arduino_Core_STM32#2260] [Libraries] [answered]
As in the subject:
I have Nucleo F746ZG, uploaded the example WebServerFreeRTOS from STM32Ethernet library and after 3..10 minutesexecution of the code hangs.

**Comment by xRadx** [comment]
non RTOS seems to run fine

**Reply by fpistm** [reply, marked as answer]
You should try to change the memory allocation implementations to use:
https://github.com/stm32duino/STM32FreeRTOS/blob/f3abddcd420bd92a862b35877181392bd2647b2a/src/FreeRTOSConfig_Default.h#L49
